rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/oracle/OCIBakeHandler.groovy

```diff
@@ -0,0 +1,131 @@
+/*
```

Comment:
Shout out to not including Groovy code in main src tree.

Reply:
What is recommended approach?  Place java code in rosco-core/src/main/groovy or create a new directory rosco-core/src/main/java?

Reply:
Placing it in `groovy` is fine.